cmd/oras/tag/tag.go

```diff
@@ -26,14 +32,15 @@ type tagOptions struct {
 	option.Common
 	option.Remote
 
-	srcRef    string
-	targetRef string
+	defaultTagNOptions oras.TagNOptions
```

Comment:
Why is this called `default`? My understanding is that this is `tagNOptions`?

Reply:
used `concurrence int64` now instead.